src/_posts/2025-08-08-employee-spotlight-adam.md

```diff
@@ -0,0 +1,58 @@
+
+### What is your most-used emoji?
+
+🤪 - I like that it's happy, whimsical, and an appropriate response to most things in this world.
+
+<img src="/assets/blog/2025/2025-08-adam-3.jpg" alt="Photo of Adam Linder." width="350">
+
+### What Compiler philosophy do you resonate with the most? Why?
+
+First and always inclusive - too often I hear non-riders say that they would take public transit if it were closer to their homes or went where they wanted to go. But they always neglect the bus system, the bus stops right outside of their homes. First and always inclusive reminds me that this is a PUBLIC service, meant to help EVERYONE, _even_ those with no other option. It's why I also believe in fareless transit, and resist the calls to ban the Unhoused from the system... These services are the lifeblood of our economy and if we forget that, we are leaving those most in need behind in favor of... a Disneyland style monorail funicular... I truly believe that public service for all is the goal here.
+
+### How have you grown professionally while on our team?
+
+Since starting at Compiler I have learned so much! My mind has exploded with transit! From all the acronyms, to even just behind the scenes work of what makes the wheels go 'round, I have appreciated everything I've learned. Transit was an interest and a hobby of mine, specifically the politics behind transit expansion, but now seeing the nuts and bolts of the technology that goes into everything once it's implemented has been eye opening.
+
+### Which accomplishments are you most proud of?
+
+In 2016, I created a map of the Los Angeles MetroRail and "expanded" the map to show the dreams of what our system could look like if the sales tax measure slated to be on the ballot would pass that Fall. I placed my image in tandem with the current network and created a "blinking gif." That gif went viral! It was written up in local newspapers and I was even interviewed by NPR in regards to the traction it was seeing. Once that sales tax measure passed, I was contacted by the then Mayor's office and asked if I'd be interested in a potential job with them. Having _just_ started on a hit television show, I turned them down... Over the next four years I learned many lessons about life and the world around me, and after all that I still ended up working in transit! Some things truly are just meant to be.
+
+<img src="/assets/blog/2025/2025-08-adam-la-metrorail-measure-m.jpg" alt="">
```

Comment:
Need alt text for this image

Reply:
"A photo of the Los Angeles MetroRail "Dream Map" created by Adam"
Photo Credit: Adam Linder